CONTRIBUTING_GUIDE.md

```diff
@@ -0,0 +1,7 @@
+# Contributing guide
+
+## DRAFT / IDEAS
+
+- **Report lack of an example / propose an improvement** --> open an issue
+- **Add an example but don't have much time** --> open a PR with only the doc added and open an issue for the code to be added (you can edit the doc and open the PR straight from github)
```

Comment:
in PR comments, add code from project, will be implemented later by someone else